docs/read_the_docs/source/workflow_inputs/characteristics.rst

```diff
@@ -3,6 +3,27 @@
 Housing Characteristics
 =======================
 
+Each parameter sampled by the national project is listed alphabetically below.
+For each, the following (if applicable) are reported based on the contents of `source_report.csv <https://github.com/NREL/resstock/blob/develop/project_national/resources/source_report.csv>`_:
+
+- **Description**
+- **Created by**
+- **Source**
+- **Assumption**
+
+Additionally for each parameter, an **Arguments** table is populated (if applicable) based on the contents of `ResStockArguments's measure.xml file <https://github.com/NREL/resstock/blob/develop/measures/ResStockArguments/measure.xml>`_:
+
+- **Name**
+- **Units**
+- **Choices**
+- **Description**
```

Comment:
Look at BuildResHPXML/measure.xml to get: either (A) **Choices** gets updated with "auto" and "<type>" or (B) **Type**, and then (C) **Required** (Yes or No, so that we can populate "auto" when it's not a choice argument). Optional here is to populate "true" and "false" in the Choices column for bool arguments.

@afontani

Reply:
I'm writing the following columns to the Arguments table, in the following order:
* Name
* Required (comes from BuildResidentialHPXML)
* Units
* Type (comes from BuildResidentialHPXML)
* Choices
  * all optional choice arguments have "auto" in addition to all other choices
  * all optional boolean arguments have "auto", "true", "false"
  * add "auto" choice if "OS-HPXML default" string in the Description and Type is String/Double/Integer
* Description